server/server.go

```diff
@@ -233,15 +250,27 @@ func New(config *Config) (*Server, error) {
 
 	utils.StartReaper()
 
-	if err := os.MkdirAll(config.ImageDir, 0755); err != nil {
+	store, err := storage.GetStore(storage.StoreOptions{
+		RunRoot:            config.RunRoot,
+		GraphRoot:          config.Root,
+		GraphDriverName:    config.Storage,
+		GraphDriverOptions: config.StorageOption,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	imageService, err := scri.GetStorageImageService(store, config.DefaultTransport)
```

Comment:
> We could. I think it would actually be cleaner in terms of dependencies to move the entirety of containers/storage/cri into this repo, but I also want it to be in a place where other projects and reuse the logic instead of having to reimplement it themselves, and I'm not sure that's within the scope of what this repo wants to be.

I agree. IMO storage library shouldn't have any dependencies on the CRI API. That actually makes it harder to use from other projects. You could have convenience APIs to create a Rootfs for an image and a container ID but higher coupling than that will make it harder to reuse.